ftgo-delivery-service/src/integration-test/java/net/chrisrichardson/ftgo/deliveryservice/domain/CourierJpaTest.java

```diff
@@ -22,6 +24,7 @@
   @Configuration
   @EnableJpaRepositories
   @EnableAutoConfiguration
+  @Import(EventuateCommonJdbcOperationsConfiguration.class)
```

Comment:
Why `@Import(EventuateCommonJdbcOperationsConfiguration.class)` .
This is internal and this test class doesn't need any of the beans it defines.

Reply:
I will check